user_sync/app.py

```diff
@@ -75,6 +75,15 @@ def process_args():
     parser.add_argument('-d', '--remove-list',
                         help='specifies the file containing the list of users to be removed. Users on this list are removeFromOrg\'ed on the Adobe side.',
                         metavar='input_path', dest='remove_list_input_path')
+    parser.add_argument('--delete-nonexistent-users',
+                        help='Causes the user sync tool to delete users that exist on the Adobe side if they are not in the customer side AD. If the account is an Adobe account, it is remove from org\'d, but not deleted.',
+                        action='store_true', dest='delete_nonexistent_users')
+    parser.add_argument('--generate-delete-list',
```

Comment:
--generate-delete-list actually isn't in the spec because it does the same thing as --generate-remove-list.  Could be a reasonable thing to add.  Thoughts?